policyengine_us/parameters/gov/states/vt/tax/income/deductions/standard/base.yaml

```diff
@@ -3,29 +3,37 @@ metadata:
   label: Vermont standard deduction
   period: year
   unit: currency-USD
+  uprating: gov.irs.uprating
```

Comment:
is this rounded?
If so we need to add some extra uprating metadata, see DC:
https://github.com/PolicyEngine/policyengine-us/pull/4306

Need to double check in legal code

Reply:
didn't find documents saying the number is rounded.

Reply:
I dont think there is rounding based on:
![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/54683182-c8dd-4202-b24b-61b2bb6da048)

Reply:
please file an issue to check on this. the numbers are rounded so we may want to check with the revenue agency to see what other part of the law defines it